lib/Cake/Utility/Security.php

```diff
@@ -63,7 +63,7 @@ public static function inactiveMins() {
  * @return string Hash
  */
 	public static function generateAuthKey() {
-		return Security::hash(CakeText::uuid());
+		return Security::hash(openssl_random_pseudo_bytes(32));
```

Comment:
Is this worth being a standalone `Security::uniqueId($length = 32)` method? I think it would be useful in general to have such a Security class method available.